### PR TITLE
added regex for splitting queries

### PR DESCRIPTION
**Pull request by aholachek**
this addresses issue #501

### CODE REVIEW

File: src/js/components/multi_params.js [hunk 1]
```diff
@@ -193,7 +193,11 @@ define(['backbone', 'underscore', 'jquery'], function(Backbone, _, $) {
       if (!resp.trim()) {
         return attrs;
       }
-      var hashes = resp.slice(resp.indexOf('?') + 1).split('&');
+      //there can be ampersands in the q parameter
+      //now using a positive lookahead that expects an = before any " characters,
+      //ex: "q=bibstem:"a&a"&sort=date+desc" will now be split into ["q=bibstem:"a&a"", "sort=date+desc"]
+
+      var hashes = resp.slice(resp.indexOf('?') + 1).split(/&(?=[^"]+\=)/);
```
Comment (romanchyla) [inline src/js/components/multi_params.js]: while this solves the given bug (it relies on kinda arbitrary assumption) 

the original problem is that the query must be wrongly encoded, look at this, the multi_params can handle the '&' well

```
q = new ApiQuery({title: 'a&a'});
q.url()
"title=a%26a"
q.load(q.url()).url()
"title=a%26a"
```

Comment (romanchyla) [inline src/js/components/multi_params.js]: which means: we need to fix the root cause of the problem (probably some widget is creating a wrong query)

**Comment by aholachek** [comment]
but if it's a widget problem, why does this url not work:

https://ui.adsabs.harvard.edu/#search/q=bibstem%3A%22a%26a%22&sort=date+desc

**Comment by romanchyla** [comment]
answer to both questions lies on line 299 ;-)

https://github.com/adsabs/bumblebee/blob/master/src/js/apps/discovery/router.js#L299

**Comment by romanchyla** [comment]
@aholachek gentle nudge

**Comment by romanchyla** [comment]
solved in #557

**Pull request closed without merging**